data/overlayfiles/motd-mlm-proxy-5/etc/motd

```diff
@@ -0,0 +1,26 @@
+SUSE Multi-Linux Manager {SUMA_VERSION} Proxy
+{OS_PRETTY_NAME} {ARCH} (64-bit)
+
+   ____
+  /@    ~-.
+  \/ __ .- |
+   // // @
+
+To set up SUSE Multi-Linux Manager ${SUMA_VERSION} Proxy, follow the
+instructions at
+https://documentation.suse.com/suma/5.0/en/suse-manager/installation-and-upgrade/container-deployment/suma/proxy-deployment-vm-suma.html.
```

Comment:
This URL (and the others below) still refer to 5.0. But as we don't know yet if the structure will stay the same and we would have to only replace the version, I'd keep it as a reminder for now.